vcf.c

```diff
@@ -4261,6 +4261,30 @@ int bcf_get_variant_type(bcf1_t *rec, int ith_allele)
     if ( rec->d.var_type==-1 ) bcf_set_variant_types(rec);
     return rec->d.var[ith_allele].type;
```

Comment:
If this was amended to be `return rec->d.var[ith_allele].type & 63;` (or some equivalent with a new macro to set the maximum legacy value of 63) then the old API would be 100% backwards compatible.

Reply:
I just force pushed this change